multi/imxrt-multi/imxrt-multi.c

```diff
@@ -258,125 +270,138 @@ static int createDevFiles(void)
 		return -1;
 	}
 
+	/* Pseudo devices */
+
+	if (mkFile(&dir, id_pseudoNull, "null", multi_port, MODE_DEFAULT) < 0) {
+		return -1;
+	}
+
+	if (mkFile(&dir, id_pseudoZero, "zero", multi_port, MODE_DEFAULT) < 0) {
+		return -1;
+	}
+
+	if (mkFile(&dir, id_pseudoFull, "full", multi_port, MODE_DEFAULT) < 0) {
+		return -1;
+	}
 
 	/* UARTs */
 
 #if UART1
-	if (mkFile(&dir, id_uart1, "uart1", common.uart_port) < 0) {
+	if (mkFile(&dir, id_uart1, "uart1", common.uart_port, MODE_DEFAULT) < 0) {
```

Comment:
I'm not sure there's a point in passing mode as an argument - it's default all the way

Reply:
Removed, but in `mkFile(..)` impl. I replaced default `0` (no-access) value with  [DEFFILEMODE](https://github.com/phoenix-rtos/libphoenix/blob/2a948484a6414a9363c6c33646d517f26d2b9d73/include/sys/stat.h#L76) (all rw).